drivers/gpio/gpio_mcux.c

```diff
@@ -32,19 +42,42 @@ struct gpio_mcux_data {
 	sys_slist_t callbacks;
 };
 
+#if defined(CONFIG_SOC_SERIES_IMXRT7XX) && defined(CONFIG_SOC_FAMILY_NXP_IMXRT)
+static uint32_t GPIO_GetInstance(GPIO_Type *base)
+{
+
```

Comment:
Please notice the coding style of gpio_mcux.c file. Please dont use CamelCase.

so GPIO_GetInstance should go into gpio_get_instance. 

But anyhow, GPIO_GetInstance looks like a function that could come from nxp_hal.

Reply:
> so GPIO_GetInstance should go into gpio_get_instance.

It is not possible. This is the SDK driver API.

Reply:
In this case it seems we are defining the function within the `gpio_mcux.c` file, right? I'm aware this function is often available within the SDK driver API, but here it does not seem like we are using the SDK API.

Reply:
Updated.